Bottom: 943bd81fe8922d43c7b6e95c9e9d86581b5b54e8
Top:    f3aa2ffc22b0870255176d871bcfa55ddeda49d5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-01 23:29:23 +0100

hid/gtk: Fix weirdness with object moves


---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 0b0b157..4534905 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -800,10 +800,10 @@ ghid_invalidate_lr (int left, int right, int top, int bottom)
   int minx, maxx, miny, maxy;
   GdkRectangle rect;
 
-  dleft = Vx ((double) left);
-  dright = Vy ((double) right);
-  dtop = Vx ((double) top);
-  dbottom = Vy ((double) bottom);
+  dleft = Vx (left);
+  dright = Vx (right);
+  dtop = Vy (top);
+  dbottom = Vy (bottom);
 
   minx = MIN (dleft, dright);
   maxx = MAX (dleft, dright);
